Runtime/Scripts/NetInterfaceIntegration.cs

```diff
@@ -93,39 +95,49 @@ public string GetIPAddress(string sourceNetInterfaceName, AddressFamily addressf
       return ipAddress;
     }
 
-    public bool HasCellular()
+    private bool testSameDefaultRoute(string iName)
     {
-      NetworkInterface[] netInterfaces = GetInterfaces();
-      bool foundByIp = false;
-
-      foreach (NetworkInterface iface in netInterfaces)
+      using (Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, 0))
```

Comment:
@lgarner  I am trying to understand this part 0 -> is ProtocolType.IP and your SocketType is UDP 

<img width="753" alt="Screen Shot 2020-09-16 at 10 48 56 AM" src="https://user-images.githubusercontent.com/24863504/93373492-43dade00-f80a-11ea-8a1d-37135c718334.png">